hnsecure/src/test/resources/application.properties

```diff
@@ -16,6 +16,7 @@ issuer = https://common-logon-dev.hlth.gov.bc.ca/auth/realms/v2_pos
 valid-receiving-facility = BC00002041,BC00002047,BC00001013
 processing-domain = D
 version = 2.1
+is-filedrops-allowed = Y
```

Comment:
This also needs to be updated to match is-filedrops-allowed in src/main/resources/application.properties

Reply:
done